pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -28,6 +32,15 @@
 @given(DATETIME_JAN_1_1900_OPTIONAL_TZ, YQM_OFFSET)
 def test_on_offset_implementations(dt, offset):
     assume(not offset.normalize)
+    # This case is flaky in CI 2024-11-04
```

Comment:
Maybe ideally open an issue about it with the reproducer and add a link to that issue here?